jobbergateapi2/tests/apps/job_scripts/test_routers.py

```diff
@@ -130,6 +131,11 @@ async def test_create_job_script(
 
     count = await database.fetch_all("SELECT COUNT(*) FROM job_scripts")
```

Comment:
I see these aren't changes in the pr, and I don't really care too much, but in light of being consistent,  why use raw sql here?

Reply:
It is just easier (and faster too) to get the count, since we are only interested in that